wagtail/contrib/modeladmin/options.py

```diff
@@ -556,12 +561,23 @@ def modify_explorer_page_queryset(self, parent_page, queryset, request):
 
     def register_with_wagtail(self):
         super().register_with_wagtail()
+        self.register_modeladmin_model_check_with_wagtail()
```

Comment:
We could avoid this override on both ModelAdmin and ModelAdminGroup by adding a `register_modeladmin_model_check_with_wagtail()` method to the base class that doesn't do anything (just `pass`)

Reply:
It might be nice to shorten the name to 'register_checks' too, as they are just regular django checks, that don't necessarily have to be about the model.